include/dungeon_global_data.h

```diff
@@ -12,9 +12,14 @@
 #define DUNGEON_MAX_WILD_POKEMON 16
 #define DUNGEON_MAX_POKEMON MAX_TEAM_MEMBERS + DUNGEON_MAX_WILD_POKEMON
 
-extern struct DungeonGlobalData *gDungeonGlobalData;
+extern struct Dungeon *gDungeon;
 
-struct DungeonGlobalData_sub
+struct DungeonLocation {
```

Comment:
this is already in `global.h`

Reply:
Oh wait, Just saw the diff moving it. Is there a reason you are moving it to here?

Reply:
I moved it from `global.h` to here.

Reply:
This feels too specific of a struct to be in something like `global.h`, so I moved it to a dungeon-specific file.